### DISCUSSION TITLE
IMHO we need the ability to fetch tags without  "fetch-depth: '0'"

### DISCUSSION THREAD

**Discussion by vitalyk-multinarity** [community/community#160424] [Actions]
In my opinion, having some flag for fetching tags  ('all-tags'?) will be nice from a performance point of view. 
What do you think?

**Comment by mike-ayres** [comment]
It looks like that's supported now, but I haven't tested it. From the main readme:

```md
# Whether to fetch tags, even if fetch-depth > 0.
# Default: false
fetch-tags: ''
```
